modelscope/models/cv/ocr_detection/preprocessor.py

```diff
@@ -45,6 +45,8 @@ def __call__(self, inputs):
             img = np.array(load_image(inputs))
         elif isinstance(inputs, PIL.Image.Image):
             img = np.array(inputs)
+        elif isinstance(inputs, np.ndarray):
```

Comment:
add docstr about np.ndarray,  shape and order

Reply:
added